apps/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -424,8 +449,11 @@ change_password(Username, OldPasswd, NewPasswd, MfaToken) ->
 %% @doc Change password from CLI.
 change_password_trusted(Username, Password) when is_binary(Username), is_binary(Password) ->
     case legal_password(Password) of
-        ok -> change_password_hash(Username, hash(Password));
-        Error -> Error
+        ok ->
+            ok = emqx_dashboard_login_lock:reset(Username),
+            change_password_hash(Username, hash(Password));
+        Error ->
+            Error
     end.
```

Comment:
Is it only possible to unlock the by changing the password through cli?

Considered: 
- the wrong password was entered by mistake due to ignoring the case lock.
- the wrong keyboard layout was configured, but the user did not find it when entering password, resulting in the wrong characters being entered (for example, qwerty -> azerty is configured).

At this time, usery may not want to change the password, just need to unlock it manually.
like: `pam_tally –user USER –reset`

Reply:
>Is it only possible to unlock by changing the password through cli?

That's the original UX requirement https://emqx.atlassian.net/browse/EEC-1160

>the wrong keyboard layout was configured

As far as I understand, the scenarios of human errors are mostly covered with a sufficient number of attempts (5 by default).